common/src/warp_runner/manager/commands/raygun_commands.rs

```diff
@@ -209,7 +227,28 @@ pub async fn handle_raygun_cmd(
             current_len,
             rsp,
         } => {
-            let r = fetch_messages_from_chat(conv_id, messaging, to_fetch + current_len).await;
+            let r = fetch_messages_from_chat(
+                conv_id,
+                messaging,
+                MessageOptions::default(),
+                to_fetch + current_len,
+            )
+            .await;
+            let _ = rsp.send(r);
+        }
+        RayGunCmd::FetchPinnedMessages {
+            conv_id,
+            to_fetch,
+            current_len,
+            rsp,
+        } => {
+            let r = fetch_messages_from_chat(
+                conv_id,
+                messaging,
+                MessageOptions::default().set_pinned(),
+                to_fetch + current_len,
```

Comment:
this is probably wrong. looks like `FetchMessages` was copy/pasted, but that command doesn't fetch all messages in the conversation. I'm guessing that `FetchPinnedMessages` actually wants to fetch all pinned messages.

Reply:
im just wondering here if we want to fetch all pinned message at once? since if someone has a lot of pinned messages (for whatever reason) this might take a while

Reply:
that could be fine for a temporary solution. 

i'm not entirely sure about this but we may want to show pinned messages in order of most recent to  least recent too. Currently the oldest message appears at the top of the list and  one has to scroll down for more recent messages.

Reply:
app will fetch up to 100 pinned message now and will have them in reverse order